Code/Mantid/Framework/PythonInterface/plugins/algorithms/ApplyNegMuCorrection.py

```diff
@@ -0,0 +1,94 @@
+from mantid.api import *  # PythonAlgorithm, registerAlgorithm, WorkspaceProperty
+
+#pylint: disable=no-init, too-many-arguments
```

Comment:
About the 'too-many-arguments' that we saw the other day, I think that this reasonable in this case. It just happens that the algorithm and some methods have many parameters.
An alternative is to create a data holder class, but then pylint will most likely complaint that your class has too many data members! So you would need to use a similar disable directive to allow it.